builtins/amp-ad.js

```diff
@@ -22,7 +22,6 @@ import {IntersectionObserver} from '../src/intersection-observer';
 import {isLayoutSizeDefined} from '../src/layout';
```

Comment:
per @cramforce, we would still like to have the resizable attribute and overflow element. So not sure if these would still be helpful for `amp-iframe`.